setup.py

```diff
@@ -36,6 +36,6 @@ def read(fname):
         "Programming Language :: Python :: 3.5",
         "License :: OSI Approved :: GNU Affero General Public License v3",
     ],
-    install_requires=["github3.py", "pygithub", "click", "requests", "jinja2"],
+    install_requires=["github3.py", "pygithub", "click", "requests", "jinja2", "pyyaml", "tabulate"],
```

Comment:
Not really sure why we need to duplicate this. These dependencies are already listed in `requirements.txt`. That file is also used for the dependencies of the snap package. Is that file not automatically used when you install it manually?

Reply:
`requirements.txt` and `setup.py` are 2 different things.
If you distribute the package PyPI, users install it via `pip install`, and that does **not** use `requirements.txt` at all. (I know, because I installed `ghtt` in a clean venv using `pip install ghtt`, and it didn't work due to missing deps)

You only need `requirements.txt` to document which packages exactly are used. For example, if you need this for reproducibility or something. I'd argue that you don't need it at all in this project. (Can you make the snap package use `setup.py`?)

In `setup.py`, you can specify versions if they are important. For example:
```
install_requires=[
   ...
   "sentry-sdk[flask]==0.14.4",
   "my-fancy-lib >=1.5.0, <2",  # allow bugfixes and extra features, but not breaking changes
   ...
]
```